src/libuv/Native.Settings

```diff
@@ -37,11 +35,13 @@
     <ClCompile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <Optimization>Disabled</Optimization>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
     </ClCompile>
   </ItemDefinitionGroup>
 
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
     <ClCompile>
+      <MinimalRebuild>false</MinimalRebuild>
```

Comment:
Why do we need this? CI is always building from a clean repo so probably it does not matter. On a dev box if a dev needs to rebuild the dll it will make things slower.